crates/tower-cmd/src/mcp.rs

```diff
@@ -103,6 +103,13 @@ struct EmptyRequest {
     common: CommonParams,
 }
 
+#[derive(Debug, Deserialize, JsonSchema)]
+struct RunRequest {
+    #[serde(flatten)]
+    common: CommonParams,
+    parameters: Option<std::collections::HashMap<String, String>>,
```

Comment:
nit: I was just thinking that it'd be good to push `use std::collections::HashMap` onto the top of the file. But, then I thought there's probably a guideline for when we should `use` something versus directly referencing it, and I'd bet it's about how many times the thing is used in a file?

So, now I'm not sure :) Just thinking out loud, no real action in this comment but perhaps it will inspire conversation elsewhere.